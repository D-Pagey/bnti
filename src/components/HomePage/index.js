import React from 'react';
import { Link } from 'react-router-dom';

export default function HomePage() {
    return (
        <div>
            <h1>Bantiiiii</h1>
            <p>API_GATEWAY = {process.env.REACT_APP_API_GATEWAY}</p>
            <Link to="/login">Go to login</Link>
        </div>
    );
}
