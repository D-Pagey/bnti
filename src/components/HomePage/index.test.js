import React from 'react';
import { render } from 'react-testing-library';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import HomePage from '.';

const renderWithRouter = (ui) => render(<Router history={createMemoryHistory()}>{ui}</Router>);

describe('HomePage component', () => {
    it('should render', () => {
        const { container } = renderWithRouter(<HomePage />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
