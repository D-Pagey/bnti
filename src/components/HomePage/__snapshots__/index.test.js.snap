// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render 1`] = `
<div>
  <h1>
    Bantiiiii
  </h1>
  <p>
    API_GATEWAY = 
  </p>
  <a
    href="/login"
  >
    Go to login
  </a>
</div>
`;
